src/ui/MultipleFilesMessageItemBody/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactElement, useState } from 'react';
+
+import Icon, { IconTypes, IconColors } from '../Icon';
+import { MultipleFilesMessage, UploadedFileInfo } from '@sendbird/chat/message';
+import ImageRenderer from '../ImageRenderer';
+import ImageGrid from '../ImageGrid';
+import FileViewer from '../FileViewer';
+
+interface Props {
+  className?: string | Array<string>;
+  message: MultipleFilesMessage;
+  isByMe?: boolean;
+  mouseHover?: boolean;
+  isReactionEnabled?: boolean;
+  truncateLimit?: number;
+}
+
+// Full width is 400px. There are two items in a row which means image width is 200px. padding is 4px and gap is 4px.
+const MULTIPLE_FILES_IMAGE_SIDE_LENGTH = 'calc(200px - 6px)';
```

Comment:
Q. how 4px came out from "padding is 4px and gap is 4px"?

Reply:
ah i knew ill get this question lol. its like below
[left padding] [left image] [gap] [right image] [right padding]

Reply:
I also can't understand well, will it be
[4px] [200px] [4px] [200px] [4px]?

Then the maximum width should be 412px

Reply:
i know this is confusing but this was the simplest way for me.
so the image grid width should be 400px fixated.
so each image's side length should have 200px including its side padding and gap.
so 200 - (4 / 2) - (4 / 2) - (4 / 2) = 200 - 6

Reply:
Or we can create a const for 4 and 2 too for the better readability. Not sure the calculation is correct as your intention but, the idea is like below. 

```
const MESSAGE_CONTAINER_WIDTH = 400;
const PADDING = 4;

const MULTIPLE_FILES_IMAGE_SIDE_LENGTH = `calc(${MESSAGE_CONTAINER_WIDTH / 2} - 3*${PADDING / 2})`
```